co2eq/purchase/footprints.yml

```diff
@@ -1,9 +1,3 @@
-# yamllint disable rule:empty-lines rule:line-length
```

Comment:
this shouldn't be deleted

Reply:
Should I just add it manually?
And what about the point I raised in the comment above?

Reply:
yes you can add it back manually. 
I didn't see the other comment sorry :) I'll reply now.